runner/consts_test.go

```diff
@@ -2,29 +2,10 @@ package runner
 
 import (
 	"testing"
-
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/metrumresearchgroup/wrapt"
 )
 
-// TODO: remove initResults if we find no use
```

Comment:
@elisarver made this comment 3 weeks ago. I think we're fine removing this, unless someone has a reason to keep it. Everything passes without it though.

Reply:
It's ok to remove this. Go compiler won't compile with missing variables, and this has been commented out to no ill effect.